kubernetes/templates/eks_cluster.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: "TorchserveCluster"
+  region: "us-west-2"
+  
+nodeGroups:
+  - name: ng-1
+    instanceType: g4dn.xlarge
```

Comment:
@dhanainme  Please add instructions for what user needs to change to customize these -- choose diff instance type, aws region etc. To make the template general pick values from env variable or add option for generating the templates

Reply:
Fixed